app/scrapers/__init__.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import print_function
```

Comment:
this helps us to use python 3 based syntax for print in query-server. This was used in the previous version of scrapper so I continued using the same print here